libs/prov/prov_k8s_cortx_deploy.py

```diff
@@ -1266,6 +1218,11 @@ def test_deployment(self, sns_data, sns_parity,
                     assert_utils.assert_true(resp[0], resp[1])
         if self.deployment_type not in self.exclusive_pod_list:
             if setup_client_config_flag:
+                LOGGER.info("Setting the current namespace")
```

Comment:
repeating here?

Reply:
if someone runs the job just to use for setup client and run IO. the above one is under deploy flag.